src/SwedbankPay/Api/Service/Paymentorder/Request/GetCurrentPayment.php

```diff
@@ -9,6 +9,6 @@ class GetCurrentPayment extends Request
     public function setup()
     {
         $this->setRequestMethod('GET');
-        $this->setRequestEndpoint('/psp/paymentorders/%s/currentpayment');
+        $this->setOperationRel('current_payment');
```

Comment:
Is `current_payment` with an underscore the correct `rel`?

Reply:
@asbjornu No, that's "virtual" `rel`. We need to get contents of some nodes. See list over there https://github.com/SwedbankPay/swedbank-pay-sdk-php/blob/feature/no-urls-building/src/SwedbankPay/Api/Service/Request.php#L297